docs/compute_resources.md

```diff
@@ -0,0 +1,173 @@
+# Compute Resources
+
+When specifying a [pod](./pods.md), you can optionally specify how much CPU and memory (RAM) each container
+needs.  When containers have resource limits, the scheduler may be able to make better decisions about
+which nodes to place pods on, and contention for resources can be handled in a consistent
+manner.
+
+
+*CPU* and *memory* are each a *resource type*.  A resource type has a base unit.
+CPU is specified in units of cores.  Memory is specified in units of bytes.
+
+CPU and RAM are collectively refered to as *compute resources*, or just *resources*.  Compute
+resources are measureable quantities which can be requested, allocated, and consumed.  They are
+distinct from [API resources](./working_with_resources.md).  API resources, such as pods and
+[services](./services.md) are objects that can be written to and retrieved from the Kubernetes API
+server.
+
+## Container and Pod Resource Limits
+Each container of a Pod can optionally specify `spec.container[].resources.limits.cpu` and/or
+`spec.contawiner[].resources.limits.memory`.  The `spec.container[].resources.requests` field
+is not currently used and need not be set.
+
+Specifying resource limits is optional.  In some clusters, an unset value may be replaced with a
+default value when a pod is created or updated.  The default value depends on how the cluster is
+configured.
+
+Although limits can only be specified on individual containers, it is convenient to talk about pod
+resource limits.
+A *pod resource limit* for a particular resource type is the sum of the resource limits of that type for each
+container in the pod, with unset values treated as zero.
+
+The following pod has two containers.  Each has a limit of 0.5 core of cpu and 100MB of memory.
```

Comment:
s/100MB/100MiB (100 * 2^20 bytes)/

(Optional; corresponds with the Mi comments below)